src/components/game/Board.jsx

```diff
@@ -1,40 +1,28 @@
-import React, { useState } from "react";
-
+import React from "react";
```

Comment:
Minor comment: no need to import this with functional components after React v17 (since you don't need other modules like hooks you can remove this line altogether).